Backend/lib/message-service/src/Lib/Domain/Types/Message/Message.hs

```diff
@@ -12,15 +12,16 @@
  the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
 -}
 
-module Domain.Types.Message.Message where
+module Lib.Domain.Types.Message.Message where
```

Comment:
As this library itself is message-service, don't create one more subfolder inside Types.